config/jobs/kubernetes/sig-autoscaling/azure/cluster-autoscaler-azure-presubmits.yaml

```diff
@@ -18,25 +18,33 @@ presubmits:
       timeout: 5h
     path_alias: k8s.io/autoscaler
     extra_refs:
-    - org: jackfrancis # change to kubernetes-sigs when fully baked
+    - org: nojnhuh # change to kubernetes-sigs when fully baked
       repo: cluster-api-provider-azure
-      base_ref: cluster-autoscaler-e2e # change to latest release branch when fully baked
+      base_ref: cas # change to latest release branch when fully baked
```

Comment:
We should be able to update this to the upstream main branch of CAPZ once https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/4946 is merged.